Makefile_test.common

```diff
@@ -33,7 +33,7 @@ $(1)/ok: $(1)/$(1).v
 
 endef
 
-diff_test = diff $(1)/$(1).golden.$(2) $(1)/$(1).$(2)
```

Comment:
The diff test should not sort.  Normalization should occur on the plugin side, so that for the same inputs, you get the same outputs. @mithro , thoughts?

Reply:
I concur that it would be better for the output sdc file to be produced in sorted order if possible.

Reply:
Added sorting to SDC output and reverted sorting in diff_test.